docs/upgrading/configuration-changes.md

```diff
@@ -80,6 +80,8 @@ proxy:
     timeout: 2
 ```
 
+Please note the default port for healthcheck is changed from `3000` to `80`. You might need to change this port back to `3000` if you are upgrading from v1.
```

Comment:
That is the port the proxy uses for all requests, so it's not healthcheck specific. Maybe we could add an example and word it something like this:

Please note that the healthchecks will use the `app_port` setting which defaults to port 80. Previously healthchecks defaulted to port 3000. You can change this back with:

```
proxy:
  app_port: 3000
```

Reply:
thanks! make sense!

updated.